Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts

```diff
@@ -126,6 +129,9 @@ async function run() {
             await DeployUsingKuduDeploy(webDeployPkg, azureWebAppDetails, publishingProfile, virtualApplication, isFolderBasedDeployment, takeAppOfflineFlag);
 
         }
+        if(scriptType) {
+            await kuduUtility.runPostDeploymentScript(publishingProfile, scriptType, inlineScript, scriptPath, takeAppOfflineFlag);
```

Comment:
If the type is scriptPath, can you upfront check for file presence and give meaningful error about file not found.